docs/content/stable/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -94,6 +94,8 @@ The type of volume provisioned for YugabyteDB Anywhere and YugabyteDB depends on
    - [Google Kubernetes Engine: persistent volumes and dynamic provisioning](https://cloud.google.com/kubernetes-engine/docs/concepts/persistent-volumes)
    - [Google Cloud: regional persistent disks](https://cloud.google.com/compute/docs/disks/high-availability-regional-persistent-disk)
 
+1. Prefer a storage class based on remote volumes like cloud provider disks instead of [local storage volumes attached directly to the kubernetes node](https://kubernetes.io/docs/concepts/storage/volumes/#local) or [local ephemeral volumes](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/). Local storage provides great performance, but the data is not replicated and can be lost if the node fails or undergoes maintenance. This can cause a full remote bootstrap of YugabyteDB data within a pod when the pod is moved to a different node during common maintenance operations like cluster node pool upgrades or maintenance operations like rolling software upgrades. [It is not recommended to use local storage volumes for production use cases](../../../deploy/kubernetes/best-practices/#local-versus-remote-ssds).
```

Comment:
```suggestion
1. Use a storage class based on remote volumes (like cloud provider disks) rather than [local storage volumes attached directly to the kubernetes node](https://kubernetes.io/docs/concepts/storage/volumes/#local) or [local ephemeral volumes](https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/). Local storage provides great performance, but the data is not replicated and can be lost if the node fails or undergoes maintenance, requiring a full remote bootstrap of YugabyteDB data within a pod. Local storage is [not recommended for production use cases](../../../../deploy/kubernetes/best-practices/#local-versus-remote-ssds).
```